### DISCUSSION TITLE
State of the 1.21 branch?

### DISCUSSION THREAD

**Discussion by zenfyrdev** [Sinytra/Connector#1274] [General]
Hi, with the release of NFFAPI, I was wondering what the state of the base repo is?

I started testing mods and there was an issue with dependencies of JIJ dependencies. Is it a known issue, or should it be reported?

Anyway, thank you for your amazing work!

**Comment by Su5eD** [comment]
Hi, please do report the issue, I'll make sure to have look. I remember running into a similar issue where nested dependencies (JIJ in JIJ) wouldn't load, and it turned out to be due to a FFAPI module's version being incorrect. Somehow, rather than displaying an error, the loader simply excluded the dependencies from being loaded. Try using the latest release of FFAPI, that particular problem has been resolved, maybe it will help you as well.

**Reply by zenfyrdev** [reply]
Will do! Btw, the silent eviction is also an issue with upstream Fabric Loader. https://github.com/FabricMC/fabric-loader/issues/868

**Reply by zenfyrdev** [reply]
Huh, the latest 1.21 build fixed the issue.
